include/struct_defs/game_options.h

```diff
@@ -0,0 +1,52 @@
+#ifndef POKEPLATINUM_STRUCT_GAME_OPTIONS_H
+#define POKEPLATINUM_STRUCT_GAME_OPTIONS_H
+
+/**
+ * Managed struct of configurable options for a game which are customizable
+ * by the player.
+ */
+typedef struct {
+    u16 textSpeed      : 4;     ///< Controls the speed at which text renders
+                                ///< in scrolling message frames. The raw value
+                                ///< must be translated into a reduction factor.
+                                ///< Supports values up to 15, but all values
+                                ///< greater than 1 are treated as equivalents.
+
+    u16 audioMode      : 2;     ///< Controls the method by which audio is
```

Comment:
Is it possible to use the enums here? Don't remember if possible to set enum type, but don't bother if not matching